.github/stale.yml

```
@@ -11,15 +11,18 @@ exemptLabels:
 # Label to use when marking an issue as stale
 staleLabel: stale
 # Comment to post when marking an issue as stale. Set to `false` to disable
-markComment: >
-  This issue has been automatically marked as stale because it has not had
-  recent activity. It will be closed if no further activity occurs.
+markComment: |
+  This issue has been marked as *stale* because it did not see any
+  activity for the last 60 days.
 
-  Thank you for your contributions.
+  *Stale* issues are closed after 14 days, unless the label is removed
+  by a maintainer or someone (anyone) comments on it.
```

what do you want to emphasize with the `(anyone)`?

```suggestion
  by a maintainer or someone comments on it.
```

---

That really anyone, not just participants or maintainers or unicorns.

But on a second read it's indeed redundant and might confuse folks. Will remove it. Thanks for pointing out!